lib/Parser/OglExtension.php

```diff
@@ -120,10 +121,56 @@ public function incorporate($other, $commit) {
         }
     }
 
+    /**
+     * Add a sub-extension, or merge it if it already exists.
+     *
+     * @param string $name Name of the extension.
+     * @param string $status Status of the extension.
+     * @param array $supportedDrivers List of drivers supported for this extension.
+     * @param #Mesamatrix\Git\Commit $commit The commit used by the parser.
```

Comment:
Unfortunately that doesn't work, PHP (and by extension, the static code analysers) require either a locally-referenced class (aka in the same directory) or an absolute class with full namespace. There is no support for C++ style sub-relative namespaces, as you suggest :smile: